lib/dialogue-agent/conversation.ts

```diff
@@ -342,13 +363,13 @@ export default class Conversation extends events.EventEmitter {
         }
     }
 
-    private async _errorWrap(fn : () => Promise<void>, platformData : PlatformData) : Promise<void> {
+    private async _errorWrap(fn : () => Promise<void>, command : string|null, platformData : PlatformData) : Promise<void> {
         try {
             try {
                 await fn();
             } catch(e) {
                 if (this._isUnsupportedError(e))
-                    await this._doHandleCommand(new UserInput.Unsupported(platformData), null, [], true);
+                    await this._doHandleCommand(new UserInput.Unsupported(command!, platformData), null, [], true);
```

Comment:
This null assertion doesn't look right? It seems from the type change you just made that `command` can indeed be null at the beginning of the function, and there is nothing that prevents it being null here.